src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -23,6 +23,7 @@ internal static partial class Library
         /// Texl Instance of <see cref="DistinctFunction"/>.
         /// </summary>
         public static readonly TexlFunction DistinctInterpreterFunction = new DistinctFunction();
+        public static readonly TexlFunction EncodeUrlFunction = new EncodeUrlFunction();
```

Comment:
>public static readonly TexlFunction EncodeUrlFunction = new EncodeUrlFunction(); [](http://example.com/codeflow?start=8&length=80)

We can remove this and add a new reference at C:\Users...\Microsoft.PowerFx.Core\Texl\BuiltinFunctionsCore.cs. Something like:
public static readonly TexlFunction IsUTCToday = _featureGateFunctions.Add(new IsUTCTodayFunction());

We might change this from featureGateFunctions to _library after chatting with PA team.

Reply:
Fixed!